package.json

```diff
@@ -0,0 +1,47 @@
+{
+  "name": "@logdna/eslint-config-logdna",
```

Comment:
I'm not sure how much value there is to it being scoped to `@logdna`

if it is published as `eslint-config-logdna` then it fits better into the eslint tooling. Like you can just reference it as `logdna` for example

```javascript
eslintConfig: {
  extends: 'logdna'
}
```
Also there are a couple package managers that hoist these patterns by default `eslint-config-*`  to facilitate effortless integration into editor tooling.

If we're going to open source it we should try to make it fit into the existing patterns and tooling exposed by eslint community as much as possible.

Reply:
Consistency?  We've published our public clients and such in that namespace.  I see no reason to not re-use it for anything in our public namespace.

Reply:
good points though, I'll change it.

Reply:
either way works and I'm ok with the latest, just noting that as `@logdna/eslint-config` we would extend it like:

```js
eslintConfig: {
  extends: '@logdna'
}
```

not a big fan of the special rules eslint has for this. I see plenty of packages doing it both ways (some even `@namespace/eslint-config-namespace`).